Brisk/Views/AttachmentDroppableView.swift

```diff
@@ -35,15 +35,21 @@ final class AttachmentDroppableView: NSView {
 }
 
 fileprivate extension NSDraggingInfo {
-    private var files: [String] {
-        return self.draggingPasteboard().propertyList(forType: NSFilenamesPboardType) as? [String] ?? []
+    private var files: [URL] {
+        let asStrings = self.draggingPasteboard().propertyList(forType: makeFileNameType()) as? [String] ?? []
```

Comment:
You can replace `makeFileNameType` inline with `.init(rawValue: "foo")` for this case IMO

Reply:
I left the makeFileNameType() in place as it is used twice and gives room for the explainatory comment. Maybe in the future somebody with more thorough macOS SDK knowledge than me could find a way to use the enum and not stringy version.